lib/grant.js

```diff
@@ -449,7 +452,24 @@ function saveRefreshToken (done) {
 }
 
 /**
- * Create an access token and save it with the model
+ * Pass the scope string to the model for saving
+ *
+ * @param  {Function} done
+ * @this   OAuth
+ */
+function saveScope (done) {
+  var scope = this.scope || this.req.body.scope;
+
+  var self = this;
+  this.model.saveScope(scope, self.accessToken, function (err, acceptedScope) {
```

Comment:
That's a very good point. I agree.

Also, :+1: for merging this into  `saveAccessToken`.